iOS/Collection View Management /CollectionViewManagerCell.swift

```diff
@@ -68,9 +69,24 @@ class CollectionViewManagerCell: UICollectionViewListCell {
 
         // Customize the background color to be clear, no matter the state.
         backgroundConfig.backgroundColor = ThemeColor.clear.color
-
+        
         // Apply the background configuration to the cell.
         self.backgroundConfiguration = backgroundConfig
+        
+        if state.isHighlighted {
```

Comment:
Needed some sort of feedback when cell is highlighted. This is super minimal but effective.